### ISSUE TITLE
Problem to create tables in Oracle

### ISSUE THREAD

**Issue by mundodojava** [javers/javers#383]
I'm using org.hibernate.dialect.Oracle10gDialect as hibernate.dialect at my hibernate.cfg.xml .
```java
private void alterCommitIdColumnIfNeeded() {
        if(this.getTypeOf("jv_commit", "commit_id") == 12) {
            logger.info("migrating db schema from JaVers 1.3.15 to 1.3.16 ...");
            if(this.dialect instanceof PostgresDialect) {
                this.executeSQL("ALTER TABLE jv_commit ALTER COLUMN commit_id TYPE numeric(12,2) USING commit_id::numeric");
            } else if(this.dialect instanceof H2Dialect) {
                this.executeSQL("ALTER TABLE jv_commit ALTER COLUMN commit_id numeric(12,2)");
            } else if(this.dialect instanceof MysqlDialect) {
                this.executeSQL("ALTER TABLE jv_commit MODIFY commit_id numeric(12,2)");
            } else if(this.dialect instanceof OracleDialect) {
                this.executeSQL("ALTER TABLE jv_commit MODIFY commit_id number(12,2)");
            } else if(this.dialect instanceof MsSqlDialect) {
                this.executeSQL("drop index jv_commit_commit_id_idx on jv_commit");
                this.executeSQL("ALTER TABLE jv_commit ALTER COLUMN commit_id numeric(12,2)");
                this.executeSQL("CREATE INDEX jv_commit_commit_id_idx ON jv_commit (commit_id)");
            } else {
                this.handleUnsupportedDialect();
            }
        }

    } 
```

**Comment by mundodojava** [comment]
Javers is getting 

```java
else if(this.dialect instanceof MsSqlDialect) {
```

**Comment by bartoszwalacik** [comment]
What is the problem exactly?  This is your code or copied from Javers?

**Comment by mundodojava** [comment]
I copied this code from Javers. The problem is: org.hibernate.dialect.Oracle10gDialect is not an instance of OracleDialect

```java

public class Oracle10gDialect extends Oracle9iDialect {

public class Oracle9iDialect extends Oracle8iDialect {

public class Oracle8iDialect extends Dialect {

```

So when I use this dialect I get an Exception

**Comment by bartoszwalacik** [comment]
But JaVers uses this enum https://github.com/javers/javers/blob/master/javers-persistence-sql/src/main/java/org/javers/repository/sql/DialectName.java 
not Hibernate Oracle10gDialect.

Could You clarify this issue?

**Comment by bartoszwalacik** [comment]
have you got ORA-00972
like here https://github.com/javers/javers/issues/388
?

**Comment by mundodojava** [comment]
Yes. That's my problem

**Comment by bartoszwalacik** [comment]
ok, closing this as a duplicte, watch https://github.com/javers/javers/issues/388

**Comment by bartoszwalacik** [comment]
fix released in 2.0.2
